src/account/operations/transfer/high_level/mint_native_token.rs

```diff
@@ -46,6 +46,9 @@ pub struct NativeTokenOptions {
     /// Maximum supply
     #[serde(rename = "maximumSupply")]
     pub maximum_supply: U256,
+    /// Foundry metadata
+    #[serde(rename = "foundryMetadata")]
+    pub foundry_metadata: Option<Vec<u8>>,
 }
 
 impl AccountHandle {
```

Comment:
Can't comment at line 67, but the example there needs to be updated with the foundry_metadata field

Reply:
Right, it's ignored so I missed it. Fixed.